### PR TITLE
shin/ch2081/update-routing-to-load-staking-pages-2

### PR DESCRIPTION
**Pull request by sarveshwar-singh**
Story:
https://app.clubhouse.io/emurgo/story/2081/update-routing-to-load-staking-pages

### CODE REVIEW

File: app/Routes.js [hunk 1]
```diff
@@ -111,26 +107,59 @@ export const Routes = (
...
+          component={(props) => <StakingSimulatorPage {...props} stores={stores} actions={actions} />}
+        />
+        <Route
+          exact
+          path={ROUTES.WALLETS.STAKE_ADVANCED_SIMULATOR}
+          component={(props) => <StakingAdvancedSimulatorPage {...props} stores={stores} actions={actions} />}
+        />
+      </>);
+  } else {
+    redirectRoute = ROUTES.WALLETS.TRANSACTIONS;
```
Comment (SebastienGllmt) [inline app/Routes.js]: PR is fine but I don't think we need to remove these anymore now that the key issues are all resolved and we have more time to ship support for this @nicarq

Comment (sarveshwar-singh) [reply app/Routes.js]: @SebastienGllmt 
Re-enabled.

File: app/i18n/locales/en-US.json [hunk 2]
```diff
@@ -307,6 +307,9 @@
   "wallet.hw.ledger.common.error.105": "Network error. Please check your internet connection.",
   "wallet.navigation.receive": "Receive",
   "wallet.navigation.send": "Send",
+  "wallet.navigation.stakeDashboard": "Dashboard",
+  "wallet.navigation.stakeAdvancedSimulator": "Advanced staking simulator",
```
Comment (nicarq) [inline app/i18n/locales/en-US.json]: @sarveshwar-singh they should be capitalized, also can you change the text to:

`Advanced Delegation Center`
`Simple Delegation Center`

Comment (sarveshwar-singh) [reply app/i18n/locales/en-US.json]: @nicarq 
Thanks, will update in upcoming PR.

**Pull request merged**